persist/fs/retriever.go

```diff
@@ -58,6 +60,7 @@ type blockRetrieverStatus int
 type newSeekerMgrFn func(
 	bytesPool pool.CheckedBytesPool,
 	opts Options,
+	fetchConcurrency int,
```

Comment:
Does this need to be passed in?  Can't the seeker mgr just read `opts.FetchConcurrency()`?